pkg/controller/controller_instance.go

```diff
@@ -67,7 +67,7 @@ func (c *controller) instanceDelete(obj interface{}) {
 		return
 	}
 
-	glog.V(4).Infof("Received delete event for ServiceInstance %v/%v; no further processing will occur", instance.Namespace, instance.Name)
+	glog.V(4).Infof(`ServiceInstance "%v/%v": Received delete event; no further processing will occur`, instance.Namespace, instance.Name)
```

Comment:
For these it sure it might be nice to do something like:
%s: Received delete event...", serviceInstanceName(instance)

func serviceInstanceName(instance *v1alpha1.ServiceInstance) {
    return fmt.Sprintf("ServiceInstance "%v/%v", instance.Namespace, instance.Name)
}

But as discussed in another PR we can do some more clever logging by different methods later.

Reply:
yeah, I totally agree, want the effect for now and we'll make it nice later